homeassistant/components/buienradar/sensor.py

```diff
@@ -186,36 +185,24 @@
     "symbol_5d": ["Symbol 5d", None, None],
 }
 
-CONF_TIMEFRAME = "timeframe"
-
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
Please keep the platform schema. We normally log a warning in `async_setup_platform` that the configuration.yaml is deprecated.